app/helpers/application_helper.rb

```diff
@@ -307,6 +307,15 @@ def name_title(args)
     end
   end
 
+  def link_to_search_value(args)
+    if args[:document][args[:field]].present?
```

Comment:
this line isn't needed, blacklight checks to make sure the field is present before running the method

Reply:
I'll update this one but I don't know where this check is done.